.travis.yml

```diff
@@ -17,6 +18,6 @@ script:
   - mkdir build
   - cd build
   - cmake -DMATERIALX_BUILD_PYTHON=ON ..
-  - cmake --build . --target install -- -j4
+  - sudo cmake --build . --target install -- -j4
```

Comment:
Unless you truly require install in the write-protected /usr area, you ought to be able to use `-DCMAKE_INSTALL_PREFIX=$HOME/someplace/you/own`, avoid the sudo, and thereby use the containerized Travis instances that are more plentiful (less likely to have a backlog) and have quicker startup time.